.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,43 @@
+name: Create Release Pull Request
```

Comment:
I notice that this is slightly different from this version in the module template. Seems like this file is missing this section. https://github.com/MetaMask/metamask-module-template/blob/main/.github/workflows/create-release-pr.yml#L45. Any reason for this? Is this something we should remove from the module template?

Reply:
huh, interesting :thinking: 

that second doesn't appear to be present in `MetaMask/controllers` either? (that's what I used as the basis for this).

Reply:
@mcmire so it would appear that `action-require-additional-reviewer` which relies on this is currently broker per: https://github.com/MetaMask/action-require-additional-reviewer/issues/44. In lieu of that I don't think we should block on this (since it sounds like it'll likely go away anyway).